modules/veteran_verification/app/controllers/veteran_verification/application_controller.rb

```diff
@@ -11,6 +12,21 @@ class ApplicationController < ::OpenidApplicationController
     # so we set that default.
     before_action { set_default_format_to_json }
 
+    def authenticate_token
+      return false if token.blank?
+
```

Comment:
Depending on which PR goes first, this `authenticate_token` method will also need to bring in the `if token.client_credentials_token?` check to skip the session & open_id user build up.

Reply:
The Client Credentials validation PR has been merged, this method will need a similar checked mentioned above.

Reply:
This has been resolved with the addition of the `token.client_credentials_token?` on line 14.